### DISCUSSION TITLE
how to show error as popup instead of in the minibuffer?

### DISCUSSION THREAD

**Discussion by dennissherb** [joaotavora/eglot#926] [Q&A] [answered]
I want to have errors/warnings pop up under the erroneous line when hovered rather than in the minibuffer, similar to how auto-completion suggestions show. Is this currently possible in eglot, and if so how do I enable it? 
I am using csharp-mode with eglot but I don't think it is specific to csharp.

**Comment by nemethf** [comment, marked as answer]
Eglot does not show auto-completion suggestions in a pop-up.  Company-mode does.  Similarly, Eglot does not show errors/warnings.  It delegates this task to flymake and eldoc.  And there's an auxiliary package to eldoc that might do what you want, it's called eldoc-box.
